### PR TITLE
Ptx 17727

### PR DESCRIPTION
**Pull request by vprabhakar-px**
fixing failed cases 

https://aetos.pwx.purestorage.com/resultSet/testSetID/241434
https://aetos.pwx.purestorage.com/resultSet/testSetID/241437

### CODE REVIEW

File: tests/basic/storage_pool_test.go [hunk 1]
```diff
@@ -6590,9 +6585,16 @@ var _ = Describe("{PoolResizeInvalidPoolID}", func() {
 
 			log.InfoD("Current Size of the pool [%s] is [%d]", poolUUID, poolToBeResized.TotalSize/units.GiB)
 
+			alertType := api.SdkStoragePool_RESIZE_TYPE_AUTO
 			// Now trying to Expand Pool with Invalid Pool UUID
 			err = Inst().V.ExpandPoolUsingPxctlCmd(*nodeDetail, invalidPoolUUID,
-				api.SdkStoragePool_RESIZE_TYPE_AUTO, expectedSize, false)
+				alertType, expectedSize, false)
+			if err != nil {
+				if strings.Contains(fmt.Sprintf("%v", err), "Please re-issue expand with force") {
```
Comment (lsrinivas-pure) [inline tests/basic/storage_pool_test.go:6593]: handle the case where err!=nil and error does not contain Please re-issue expand with force

File: tests/basic/storage_pool_test.go [hunk 2]
```diff
@@ -6608,32 +6610,39 @@ var _ = Describe("{PoolResizeInvalidPoolID}", func() {
 			// Now trying to Expand Pool with Invalid Pool UUID
 			err = Inst().V.ExpandPoolUsingPxctlCmd(*nodeDetail, poolUUID,
 				api.SdkStoragePool_RESIZE_TYPE_AUTO, expectedSize, false)
-			log.FailOnError(err, "Failed to resize pool with UUID [%s]", poolToBeResized.Uuid)
+			if err != nil {
+				if strings.Contains(fmt.Sprintf("%v", err), "Please re-issue expand with force") {
+					err = Inst().V.ExpandPoolUsingPxctlCmd(*nodeDetail, poolUUID,
+						api.SdkStoragePool_RESIZE_TYPE_AUTO, expectedSize, true)
```
Comment (lsrinivas-pure) [inline tests/basic/storage_pool_test.go:6616]: same as above

**Comment by vprabhakar-px** [comment]
https://aetos.pwx.purestorage.com/resultSet/testSetID/242150 : latest run with fix

**Comment by vprabhakar-px** [comment]
latest job run status : https://aetos.pwx.purestorage.com/resultSet/testSetID/244985

**Comment by vprabhakar-px** [comment]
https://aetos.pwx.purestorage.com/resultSet/testSetID/246012 is the latest run results

**Review by lsrinivas-pure — APPROVED** [review]
LGTM

**Pull request merged**